.circleci/config.yml

```diff
@@ -17,6 +17,7 @@ workflows:
       - architect/push-to-docker:
           name: push-to-quay
           image: "quay.io/giantswarm/prow-log-aggregator"
+          context: architect
```

Comment:
Is this required? I don't see it in `aws-operator`.

Reply:
Older projects have their env variables set individually.
We should be using this context to share variables in the future.